userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -62,6 +66,24 @@ The theme has features such as gradient backgrounds (`$enable-gradients`) and
 shadows (`$enable-shadows`) enabled by default. These can also be toggled in
 your project variables file by setting the variables to `false`.
 
+To add colors to or modify Bootstrap's [color maps], use
+**`assets/scss/_variables_project_after_bs.scss`**. For example:
+
+```scss
+$custom-colors: (
+  "my-favorite-color": purple
+);
```

Comment:
```suggestion
$custom-colors: (
  "my-favorite-color": purple,
  "my-other-color": pink
);
```
It might be only me, but I instinctively tried to use semicolons when I added a second color, which cause a weird 'unclosed parenthesis error', so I'd include this multi-line example.

Reply:
Great idea, thanks!